src/alph_puzzle.c

```diff
@@ -311,30 +708,30 @@ static int AlphPuzzle_CheckInput(AlphPuzzleData *data) {
     AlphPuzzle_UpdateSelectedTile(data, tileIndex, TRUE);
     PlaySE(SEQ_SE_GS_SEKIBAN_SENTAKU);
     data->unk4 = 1;
-    return ALPH_PUZZLE_STATE_2;
+    return ALPH_PUZZLE_STATE_PICKUP_TILE;
 }
 
-static int ov110_021E5CCC(AlphPuzzleData *data) {
+static int AlphPuzzleMainSeq_PickupTile_impl(AlphPuzzleData *data) {
```

Comment:
not a huge fan of the _impl superscript but at the same time GF really did just make a function that solely just calls another function for each of these huh